### DISCUSSION TITLE
PPP cookbook - recent Input Data Products

### DISCUSSION THREAD

**Discussion by ThemisTsnt** [groops-devs/groops#204] [General]
Hello,

If I want to use a configuration similar to PPP-Kinematic cookbook with data products (ΒΙΑ,OBX,SP3,CLK) which correspond to more recent dates than the end of 2020, I am obligated to use only repro3 from centers other than TUG or IGS, or is there any other available repository except CDDIS or TUG which I could use? Could GROOPS work also with not repro 2 products?  I tried to run with GFZ data products for some dates in 2022 and it couldn't recognize the biases and orbex files from this centre.
Thank you in advance for the great work that you have done :) 

Cheers

**Comment by zhedumi** [comment]
Hi,

Yeah that should not be a problem you just have to prepare the data with the respective conversion program. Within the cookbook and the example scenario there is a 01groopsConvert.xml that converts these external files. (See [cookbook Data preparation](https://groops-devs.github.io/groops/html/cookbook.gnssPpp.html) )
I mean CDDIS has all the products from all analysis centres such as GFZ, JPL, ESA etc. 
GROOPS can work with any product if the data is converted to the groopsdata. For example for orbits the broadcast, rapid or final IGS products are usable without issue.
Could you post the error or the data source you are using which lead to the error because I have used other orbits from other analysis centres such as COD without issue so I am not 100% sure where the error could be.

best regards
Patrick

**Reply by ThemisTsnt** [reply]
Dear Patrick,

Thank you so much for you help. As it turned the observation file of the specific date for the station that I was checking had an issue. I performed PPP Kinematic calculations using the respective cookbook configuration using GFZ precise products, but not repro 3 as the TUG are in the example that you provide. It worked fine with no errors and I managed to get also receiver clock biases with a little change in the GNSSProcessing Program.

Again your answer helped me a lot, because I was not sure if the different data products was the issue. 

Kind regards,
Themis
